src/crimson/osd/ops_executer.h

```diff
@@ -151,6 +151,7 @@ class OpsExecuter : public seastar::enable_lw_shared_from_this<OpsExecuter> {
     virtual ~effect_t() = default;
   };
 
+  Ref<PG> pg; // for the sake of the ObjClass API
```

Comment:
better of being more specific. like

> for `cls_current_version()`.

also `ObjClass` is a little bit misleading, reader is tempted to grep `ObjClass` at seeing it. but there is no such a type named `ObjClass`.

Reply:
also, is it okay to keep a reference to `pg_info_t` or just the  value of `last_user_version` ?

Reply:
The reference to pg is needed for other methods inside objclass.cc that are not yet implemented, therefore I don't think keeping a reference to `pg_info_t` or the value of `last_user_version` will be enough. It can work for this PR but we will have to change it back in the future anyway, is what I am saying. That's why I also want the comment to be more generic because the reference is there not solely for `cls_current_version()`, but I also agree that "ObjClass API" is misleading, any suggestions for something better?

Reply:
object class. see https://docs.ceph.com/en/latest/rados/api/objclass-sdk/

Reply:
Fixed and repushed